sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestHelper.scala

```diff
@@ -41,13 +41,12 @@ trait SQLQueryTestHelper {
       .replaceAll(
         s"Location.*$clsName/",
         s"Location $notIncludedMsg/{warehouse_dir}/")
-      .replaceAll(s"file:.*$clsName", s"Location $notIncludedMsg/{warehouse_dir}")
+      .replaceAll(s"file:.*?$clsName", s"Location $notIncludedMsg/{warehouse_dir}")
```

Comment:
```suggestion
      .replaceAll(s"file:.*?$clsName", s"file:$notIncludedMsg/{warehouse_dir}")
```

Reply:
not related to this PR but it seems a mistake to not retain `file:`

Reply:
You are right. From the context, this should be `file:`